torch_em/multi_gpu_training.py

```diff
@@ -33,37 +33,82 @@ def _create_data_loader(ds_callable, ds_kwargs, loader_kwargs, world_size, rank)
     return loader
 
 
+class DDP(DistributedDataParallel):
```

Comment:
Can you explain why this is necessary?

Reply:
Ah yes, to access the original model's functionalities (say `self.model.image_embeddings_oft` for instance), the calls from `DDP` wrapped models are `self.model.module.image_embeddings_oft`. I decided to generalize this via another class to automate this, rather changing it on the model-level calls.

Reply:
I see! That makes sense. This is a good design! Can you add a short comment on this?

Reply:
Thanks! Added a docstring to the wrapper now!